src/CurlToCSharp.IntegrationTests/RequestsTests.cs

```diff
@@ -100,8 +102,8 @@ private static Task<string> ExecuteCurlRequestAsync(string curlArguments)
 
     private static async Task<string> ExecuteCsharpRequestAsync(string curlArguments)
     {
-        var commandLineParser = new CommandLineParser(new ParsingOptions(int.MaxValue));
-        var converterService = new ConverterService();
+        var commandLineParser = new Parser(new ParsingOptions(int.MaxValue));
```

Comment:
Let's call these classes `CurlParser` and `CurlConverter`.
Parse and converter are too generic.

Reply:
Done